pinot-core/src/main/java/org/apache/pinot/core/common/datatable/DataTableBuilder.java

```diff
@@ -96,6 +99,17 @@ public DataTableBuilder(DataSchema dataSchema) {
     _rowSizeInBytes = DataTableUtils.computeColumnOffsets(dataSchema, _columnOffsets);
```

Comment:
This won't be correct because we want to fix the float value size (should be 4 but use 8 bytes in v2)

Reply:
As discussed offline, we wanner this change focus on metadata change, I will send a separate PR to bump up version to V4, which is dedicated to address all TODOs in DataTableBuilder, including:
* fix the float value size issue 
* Store bytes as variable size data instead of String
* Use one map of "String->Int" for all columns, instead a one map for one column.